pkg/config/settings/runtime_setting_log_level.go

```diff
@@ -11,43 +11,66 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util/log"
 )
 
+type LogLevelSource string
+
+const (
+	LogLevelSourceCLI     LogLevelSource = "cli"
+	LogLevelSourceRC      LogLevelSource = "remote-config"
+	LogLevelSourceDefault LogLevelSource = "default"
+	LogLevelSourceUnknown LogLevelSource = "unknown"
+)
+
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 Nitpick:** ‏As these are used in every runtime setting, let's rename these without the `LogLevel` prefix (maybe use `SettingSource` instead?) and move them to runtime_setting.go